linalg/densemat.cpp

```diff
@@ -1941,12 +1937,7 @@ void Add(const DenseMatrix &A, const DenseMatrix &B,
 void Add(double alpha, const double *A,
          double beta,  const double *B, DenseMatrix &C)
 {
-   const int m = C.Height()*C.Width();
-   double *C_data = C.GetData();
-   for (int i = 0; i < m; i++)
-   {
-      C_data[i] = alpha*A[i] + beta*B[i];
-   }
+   kernels::Add(C.Height(), C.Width(), alpha, A, beta, B, C.Data());
```

Comment:
This was a bit confusing to me, `kernels::Add` actually does not compute on device, was there a reason to replace the current code with this one?

Reply:
It's now a `MFEM_HOST_DEVICE` function, that can be used from kernels (`fem/tmop/tmop_pa_p3.cpp`)?